REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.7
 PDMats 0.9.0
-StatsFuns 0.3.1
+StatsFuns 0.7.1
```

Comment:
why require up to patch number? Why not 0.7?

Reply:
0.7.1 was the release with the necessary changes (in hindsight it should have been a minor release, I apologise).

Reply:
ok, we can let everything so for this release, in a next one it would be good to remove patch from the other packages requirements